### PR TITLE
main-banner is updated to  version 1.2

### PR DESCRIPTION
**Pull request by ayabaknu**
#### Background Information
making the component texts and images Generic

#### Impact
Improving components to make them more reusable using placeholder and mentioning the right version.

#### Issue
Solves: https://github.com/team-refcode/refcode.org/issues/163

### CODE REVIEW

**Comment by brentonstrine** [comment]
Hi Tariku, I am reviewing this now. I am moving ticket #163 into the "PRs in review" column since that ticket isn't done until this code is merged in.

image

File: components/Molecule/main-banner.html [hunk 1]
```diff
@@ -20,35 +19,33 @@
...
         }
 
-        ul li {
+        .main-banner-nav-list li {
             list-style: none;
             display: inline-block;
             padding: 20px;
             font-size: 20px;
-            font-family: "Times New Roman", Times, serif;
+            font-family: 'Lexend Deca'
```
Comment (brentonstrine) [inline components/Molecule/main-banner.html:36]: @ayabaknu can you add `sans-serif` as a fallback font? That way if Lexend Deca is not available, the font will go to sans serif. 

`font-family: 'Lexend Deca', sans-serif`

**Review by brentonstrine — CHANGES_REQUESTED** [review]
Hi @ayabaknu ! Just one minor change requested before you merge!

Also, I noticed that there is a merge conflict happening. I haven't looked at it, but you will need to resolve it before merging.

**Comment by brentonstrine** [comment]
I looked at the merge conflict. What happened is that some of my changes from PR https://github.com/team-refcode/refcode.org/pull/165 are conflicting with yours. 

This happened because my changes merged in but your local repository did not get updated with those changes before you started working. 

There is actually quite a lot of overlap between our PRs, so this might be tricky to resolve. I think the first thing you need to do is pull down the latest changes from "upstream".  This can be done either via the git command line or GitHub Desktop. I will reach out to you on Slack to talk through it.

**Comment by ayabaknu** [comment]
ok. I will add the sans-serif. I am now going to try to solve the conflict

File: components/Molecule/main-banner.html [hunk 2]
```diff
@@ -106,6 +103,7 @@
     </style>
     <markup>
       <header class="main-banner">
+      <link href="https://fonts.googleapis.com/css2?family=Lato&family=Lexend+Deca&display=swap" rel="stylesheet">
```
Comment (brentonstrine) [inline components/Molecule/main-banner.html:106]: For this to work in the pattern library, this needs to go inside an element called `<includes>` below `<markup>`:

```
<markup>
...
</markup>
<includes>
    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Lexend+Deca&display=swap" rel="stylesheet">
</includes>
```

That's just the way I built the pattern library to work!

File: components/Molecule/main-banner.html [hunk 3]
```diff
@@ -6,7 +6,6 @@
```
Comment (brentonstrine) [inline components/Molecule/main-banner.html:6]: I think this is at version 1.3 now!

**Review by brentonstrine — APPROVED** [review]
Tariku, it looks good! One other thing, you should copy/paste this to replace version 1.2 in the pattern library!

File: components/Molecule/main-banner.html (hunk 2 above)
Comment (ayabaknu) [reply components/Molecule/main-banner.html:106]: ok. I will fix that now.

File: components/Molecule/main-banner.html (hunk 3 above)
Comment (ayabaknu) [reply components/Molecule/main-banner.html:6]: ok. does every change needs 0.1 increament? should I commit now as 1.4?

**Comment by ayabaknu** [comment]
ok. I will include it in pattern library. and should I change its version? does every change need 0.1 increment?

**Comment by brentonstrine** [comment]
Great question about the CSS link. You don't need to put it in the top, putting it in the <includes> within the component is enough. 

But it doesn't hurt to put it in the top either!

**Comment by ayabaknu** [comment]
Ok.

**Pull request merged**